mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -76,19 +98,28 @@ impl<S> StoreMqttEventHandler<S> {
         })
     }
 
-    fn update_subscribed(&mut self, sub: &str) {
+    fn handle_subscribed(&mut self, sub: &str) {
         if let Some(mapper) = self.topic_mappers_updates.get(sub) {
             self.topic_mappers.insert(sub.to_owned(), mapper);
         } else {
             warn!("unexpected subscription ack for {}", sub);
         };
     }
 
-    fn update_unsubscribed(&mut self, sub: &str) {
+    fn handle_unsubscribed(&mut self, sub: &str) {
         if self.topic_mappers.remove(sub).is_none() {
             warn!("unexpected subscription/rejected ack for {}", sub);
         };
     }
+
+    fn handle_rejected(&mut self, sub: SubscribeTo) {
+        self.topic_mappers.remove(&sub.topic_filter);
+        if let Some(sender) = &mut self.retry_sub_send {
+            if sender.send(sub).is_err() {
+                warn!("unable to schedule subscription retry. channel closed");
```

Comment:
Can we log the err too?

Reply:
in this case the error is the `sub` itself. the only way error occurred is when the channel is closed https://docs.rs/tokio/1.0.2/tokio/sync/mpsc/struct.UnboundedSender.html#method.send